NEXT_CHANGELOG.md

```diff
@@ -127,6 +127,25 @@ There is a new metric called `apollo_router_processing_time` that measures the t
 
 By [@Geal](https://github.com/geal) in https://github.com/apollographql/router/pull/2371
 
+### APQ support in subgraph requests ([PR #2284](https://github.com/apollographql/router/pull/2284))
+
+Automatic persisted queries can now be used for subgraph requests. It is disabled by default, and can be configured for all subgraphs or per subgraph:
+
+```yaml title="router.yaml"
+supergraph:
+  apq:
+    subgraph:
+      # override for all subgraphs
+      all:
+        enabled: false
+      # override per subgraph
+      subgraphs:
+        products:
+          enabled: true
+```
+
+By [@Geal](https://github.com/geal) in https://github.com/apollographql/router/pull/2284 and https://github.com/apollographql/router/pull/2418
+
```

Comment:
@Geal: #2439 is about to auto-merge.   Once it does, this might merge conflict.  Could you position this CHANGELOG entry:

- _After_ "Measure the Router's processing time"
- _Before_ "Allow the disabling of automated persisted queries"

(It's almost there right now.)

Reply:
done in 833db61f